src/web/lib/messagePicker.ts

```diff
@@ -0,0 +1,103 @@
+import { record } from '@root/src/web/browser/ophan/ophan';
+
+export type MaybeFC = React.FC | null;
```

Comment:
Is this alias offering us enough value to justify its existence? Would the code be clearer if we wrote `React.FC | null` everywhere this appears?

Reply:
Yeah agreed I think it's marginal. The reason for it is that there are types later on which build on top of `MaybeFC` which to me look slightly unwieldy when we expand `MaybeFC` i.e.:

`type ShowMessage = (meta?: any) => MaybeFC;` vs `type ShowMessage = (meta?: any) => React.FC | null;`

and

`Promise<() => MaybeFC>` vs `Promise<() => React.FC | null>`

Definitely subjective though.

Reply:
Discussed offline that we think we can remove the `null` part of this type completely. It's this way because the CMP rendering exists outside of React's world, and doesn't return a component. However, returning a no-op component from the CMP show method would allow us to simplify the type, which is probably worth doing (in a separate PR though).